console/src/components/Services/Data/TableModify/ColumnEditorList.js

```diff
@@ -104,6 +105,31 @@ const ColumnEditorList = ({
 
       const isOk = getConfirmation(confirmMessage, true, colName);
       if (isOk) {
+        const pk = [];
+        let flag = true;
+        for (const [_, value1] of tableSchema.primary_key.columns.entries()) {
+          for (const [index2, value2] of tableSchema.columns.entries()) {
+            if (value2.column_name.localeCompare(colName) == 0) {
```

Comment:
2 Notes here:
 - Preferrable to use `===` over `==`
 - the whole conditional can be replaced with `if (!value2.column_name.localeCompare(colName))`, since `0` is a `falsy` value. (Read more here: https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

Reply:
Okay